api/models/User.js

```diff
@@ -31,4 +40,165 @@ userSchema.statics.create = function(name) {
 	});
 };
 
-module.exports = mongoose.model('User', userSchema);
+/**
+ * Read and Retrieve a User object from the database
+ * @param {string} user's googleID (as hash)
+ * @returns {User} one User object with matching ID
+ * @example
+ * User.read('someGoogleIdHash')
+ *     .then(user => console.log(user))
+ *	   .catch(error => console.error(error));
```

Comment:
Ditto.

Reply:
This is strange since this is fine on my machine